lib/manager/containers/LabelAssigner.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import { updateFeedSource } from '../actions/feeds'
+import LabelAssigner from '../components/LabelAssigner'
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+  }
+}
+
+const mapDispatchToProps = {
+  updateFeedSource
+}
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(LabelAssigner)
```

Comment:
In practice with newer components, I believe we've abandoned the separation of components into connected containers and the components, so it's fine to combine this in just the component file.

Reply:
Refactored in a1bc3e9